File: Python/que_5.py
# Write a program to check value is present in the list or is not

def isPresent(list, value):
    if value in list:
        return True
    else:
        return False
        
size = int(input("Enter the size of the list: "))

element = [int(input(f"Enter the {i + 1} element: ")) for i in range(size)]

value = int(input("Enter the value you want to check: "))

result = isPresent(element, value)

print(f"Is value present in list: {result}")

File: Dart/que_5.dart
import 'dart:io';

bool checkElement(List list, int check) {
  for (int i = 0; i <= list.length; i++) {
    if (list[i] == check) {
      return true;
    } else if (list[i] != check) {
      return false;
    }
  }
  return false;
}

void main() {
  print("Enter the size of the list:");
  int size = int.parse(stdin.readLineSync()!);
  List list = List.generate(size, (generator) {
    print("Enter the elements: ");
    int value = int.parse(stdin.readLineSync()!);
    return value;
  });
  print("Enter the value you want to check?");
  int check = int.parse(stdin.readLineSync()!);
  bool isTrue = checkElement(list, check);
  print("Is element exist: $isTrue");
}
